src/components/PrincipalSection/index.tsx

```diff
@@ -1,10 +1,52 @@
+import Diablo4 from '~/../../public/assets/logos/diablo-4.png'
+import HearthStone from '~/../../public/assets/logos/hearthstone.png'
+import WoW from '~/../../public/assets/logos/world-of-warcraft.png'
+import Diablo1 from '~/../../public/assets/logos/diablo-1.png'
+import StarCraft2 from '~/../../public/assets/logos/star-craft-2.png'
+import Diablo4XL from '~/../../public/assets/logos/diablo-4xl.png'
+import ThumbDiablo4 from '~/../../public/assets/banner/thumb-diablo4.png'
+
 import { Navbar } from 'components'
 
 const PrincipalSection = () => {
   return (
     <div className="min-h-[738px] bg-1 ">
       <Navbar />
-      <div></div>
+      <div className="flex min-h-[738px] items-center justify-center">
+        <ul className="flex flex-col gap-y-5">
+          <li tabIndex={0}>
+            <img src={Diablo4} alt="Diablo 4" width="48" />
+          </li>
+          <li tabIndex={0}>
+            <img src={HearthStone} alt="HearthStone" width="48" />
+          </li>
+          <li tabIndex={0}>
+            <img src={WoW} alt="World of Warcraft" width="48" />
+          </li>
+          <li tabIndex={0}>
+            <img src={Diablo1} alt="Diablo 1" width="48" />
+          </li>
+          <li tabIndex={0}>
+            <img src={StarCraft2} alt="Star Craft 2" width="48" />
+          </li>
+        </ul>
+        <div className="ml-44 mr-64 w-full max-w-[562px] font-poppins font-medium	text-white">
+          <h1 className="mb-4 text-6xl font-bold leading-[71px]">
+            Retorne à escuridão com o game Diablo IV
+          </h1>
+          <p className="mb-8 text-lg font-normal">
```

Comment:
**suggestion:** você definiu `font-medium` no pai desses 2 elementos e acabou especificando pesos diretamente neles, nesse caso eu acho que seria melhor apenas deixar diretamente nos elementos

Reply:
The font-normal is default value, right? Is more interesting didn't specify the normal on the elements?

Reply:
It depends, but in this case, you are using 3 classes where you only need 1 (on the heading). The parent `font-medium` is also affecting the button, but it shouldn't (any font weight definition for a button component should be INSIDE the component)